_data/events.json

```
@@ -32,6 +32,22 @@
             "videoURL": "",
             "buttonText": "More info"
         },
+        {
+            "date": "March 5-7, 2024",
+            "day": "Addis Ababa, Ethiopia",
+            "time": "",
+            "expire": "2024-03-08T00:00:00.0000-00:00",
+            "location": "",
+            "url": "https://connect.mosip.io/",
+            "eventType": "MOSIP Connect 2024",
+            "eventName": "WebSub as an Enabler: Facilitating Integration",
+            "presenterTwitter": "",
+            "presenter": "Presenters: Shafreen Anfar and Ayesh Almeida",
```

```suggestion
            "presenter": "Shafreen Anfar and Ayesh Almeida",
```

We don't usually add the word `Presenters`.